frontend/app_flowy/lib/workspace/domain/image.dart

```diff
@@ -8,9 +8,9 @@ AssetImage assetImageForViewType(ViewType type) {
 }
 
 extension SvgViewType on View {
-  Widget thumbnail() {
+  Widget thumbnail(Color iconColor) {
```

Comment:
Using the optional params would be better?

`Widget thumbail({Color? iconColor})`